pkg/operator/collection.go

```diff
@@ -198,259 +193,55 @@ func (r *collectionReconciler) ensureCollectorSecrets(ctx context.Context, spec
-						Args: []string{
-							fmt.Sprintf("--config-file=%s", path.Join(configDir, configFilename)),
-							fmt.Sprintf("--config-file-output=%s", path.Join(configOutDir, configFilename)),
-							fmt.Sprintf("--reload-url=http://localhost:%d/-/reload", r.opts.CollectorPort),
-							fmt.Sprintf("--listen-address=:%d", r.opts.CollectorPort+1),
-						},
-						// Pass node name so the config can filter for targets on the local node,
-						Env: []corev1.EnvVar{
-							{
-								Name: monitoringv1.EnvVarNodeName,
-								ValueFrom: &corev1.EnvVarSource{
-									FieldRef: &corev1.ObjectFieldSelector{
-										FieldPath: "spec.nodeName",
-									},
-								},
-							},
-						},
-						Ports: []corev1.ContainerPort{
-							{Name: "cfg-rel-metrics", ContainerPort: r.opts.CollectorPort + 1},
-						},
-						VolumeMounts: []corev1.VolumeMount{
-							{
-								Name:      configVolumeName,
-								MountPath: configDir,
-								ReadOnly:  true,
-							}, {
-								Name:      configOutVolumeName,
-								MountPath: configOutDir,
-							},
-						},
-						Resources: corev1.ResourceRequirements{
-							Requests: corev1.ResourceList{
-								corev1.ResourceCPU:    *resource.NewScaledQuantity(5, resource.Milli),
-								corev1.ResourceMemory: *resource.NewScaledQuantity(16, resource.Mega),
-							},
-							// Set sane default limit on CPU for config-reloader.
-							Limits: corev1.ResourceList{
-								corev1.ResourceCPU:    *resource.NewScaledQuantity(100, resource.Milli),
-								corev1.ResourceMemory: *resource.NewScaledQuantity(32, resource.Mega),
-							},
-						},
-						SecurityContext: minimalSecurityContext(),
-					},
-				},
-				Volumes: []corev1.Volume{
-					{
-						Name: storageVolumeName,
-						VolumeSource: corev1.VolumeSource{
-							EmptyDir: &corev1.EmptyDirVolumeSource{},
-						},
-					}, {
-						Name: configVolumeName,
-						VolumeSource: corev1.VolumeSource{
-							ConfigMap: &corev1.ConfigMapVolumeSource{
-								LocalObjectReference: corev1.LocalObjectReference{
-									Name: NameCollector,
-								},
-							},
-						},
-					}, {
-						Name: configOutVolumeName,
-						VolumeSource: corev1.VolumeSource{
-							EmptyDir: &corev1.EmptyDirVolumeSource{},
-						},
-					}, {
-						// Mirrored config secrets (config specified as filepaths).
-						Name: secretVolumeName,
-						VolumeSource: corev1.VolumeSource{
-							Secret: &corev1.SecretVolumeSource{
-								SecretName: CollectionSecretName,
-							},
-						},
-					},
-				},
-				ServiceAccountName:           NameCollector,
-				AutomountServiceAccountToken: ptr(true),
-				PriorityClassName:            r.opts.PriorityClass,
-				SecurityContext:              podSpecSecurityContext(),
-			},
-		},
-	}
-	// DNS policy should be set explicitly when using hostNetwork.
-	if r.opts.HostNetwork {
-		ds.Template.Spec.HostNetwork = true
-		ds.Template.Spec.DNSPolicy = "ClusterFirstWithHostNet"
-	}
```

Comment:
don't see these in the new collector manifest, just checking that's intentional

Reply:
Yeah - we don't have a reason to run on the host network anymore. We used to provide the option via the operator, but with collectors being broken out to their own manifests, users can add that themselves if they need it.